### PR TITLE
feat: More endpoint features

### LINKED ISSUE #156: [BE] Add deleteMany and just Metadata endpoints

### PR DESCRIPTION
**Pull request by Corgam**
Closes #156 
- Added option for getAll and getFiltered endpoints to return just metadata
- Added endpoint for deleteMany
- Updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,39 +11,56 @@ Authors:
 - Florian Jäger
 - Frederik Stalschus
 
-## Project setup
+## Project Deployment
 
 1. Install [Docker](https://docs.docker.com/engine/install/) and run the docker daemon.
```
Comment (Fr3574) [inline README.md:16]: Should we also require that docker compose is installed?

File: README.md [hunk 2]
```diff
@@ -11,39 +11,56 @@ Authors:
...
-Additionally, BE uses types specified in the `common/types` folder.
+### Backend structure
 
-- `config` contains all config variables used by the BE code
-- `controllers` includes all routes logic and routing code
-- `errors` contains declaration of custom errors, used by the error middleware
-- `middlewares` contains all middleware layers used by the express server
-- `models` contains schemas for the databases
-- `services` contains the algorithms that fulfill the business requirements
+The BE is build with dockerized, microservices, layered RESTful (model, service, and controller) architecture in mind,thus more microservices can be added easily by deploying further Docker containers.
```
Comment (Fr3574) [inline README.md:45]:
```suggestion
The BE is build with a dockerized, RESTful layered (model, service, and controller), microservices architecture in mind, thus more microservices can be added easily by deploying further Docker containers.
```

**Review by Fr3574 — CHANGES_REQUESTED** [review]
Could you please also add a test for `...&onlyMetadata=true`?

File: README.md (hunk 1 above)
Comment (Corgam) [reply README.md]: Done.

**Comment by Corgam** [comment]
Due to the time limit, I will not focus on the tests. I have added them as the backburner issue #145.

**Review by Fr3574 — APPROVED** [review]
LGTM

**Pull request merged**